api/types.go

```diff
@@ -0,0 +1,141 @@
+package api
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
+)
+
+// LoadBalancerList is a collection of LoadBalancer
+type LoadBalancerList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+	Items           []LoadBalancer `json:"items"`
+}
+
+// +genclient=true
+
+// LoadBalancer describe a LoadBalancer witch providers Load balancing for services
```

Comment:
Can u add more detains on at least the following concepts here as well: proxy, provider.